src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableForm.vue

```diff
@@ -107,10 +114,16 @@ const {
     },
     setForm,
     invalidState,
+    invalidTexts,
 } = useFormValidator({
     name: '',
 }, {
-    name(value: string) { return value.trim().length > 0; },
+    name(value: string) {
+        if (props.variableNames.includes(value)) {
+            return i18n.t('Name is duplicated');
+        }
+        return value.trim().length > 0;
```

Comment:
nitpick: Invalid text seems to be needed here too.

Reply:
I add annotation! Thank you!